src/main/java/filters/ExcludeTimeFilter.java

```diff
@@ -0,0 +1,61 @@
+package filters;
+
+import entities.Schedule;
+import entities.Section;
+import entities.Timeslot;
+
+import java.sql.Time;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
```

Comment:
This description makes sense with the old way Filters worked, but now it doesn't. I like the general idea tho